### ISSUE TITLE
Depthwise + 1x1 pointwise convolution 

### ISSUE THREAD

**Issue by zamir1001** [oneapi-src/oneDNN#1062]
Hi,

Is it possible to perform a (fused?) depthwise convolution followed by a pointwise 1x1 convolution during training by using the 
DNNL_ARG_ATTR_POST_OP_DW post-op? And should the DNNL_ARG_ATTR_POST_OP_DW point to the same memory as DNNL_ARG_DST ?

thanks

**Comment by vpirogov** [comment]
Hi @zamir1001.

Fusion for convolution with 1x1 kernel and depthwise convolution is supported only for inference. See `Depthwise Post-op` paragraph in [Post-ops](https://oneapi-src.github.io/oneDNN/dev_guide_attributes_post_ops.html) section of oneDNN developer guide. 

For training oneDNN will need intermediate results to compute data and weight gradients, so the fusion is not useful from the performance perspective. If you are implementing neural network training you'll need to compute these two convolutions as separate operations. See an example implementing depthwise convolution with oneDNN in https://github.com/oneapi-src/oneDNN/issues/926.

**Comment by zamir1001** [comment]
Hi @vpirogov ,

Thanks a lot for all the links and clarifications!
It's only used in optimizing inference.
